language/move-lang/functional-tests/tests/libra/natives/event_no_resource.move

```diff
@@ -0,0 +1,15 @@
+module ResourceHandle {
+    use 0x1::Event;
+    use 0x1::LBR::LBR;
+    use 0x1::Libra;
+
+    resource struct T {
+        foo: Event::EventHandle<Libra::Libra<LBR>>
+    }
+
+    public fun new(): T {
+        let handle = Event::new_event_handle<Libra::Libra<LBR>>();
+        return T { foo: handle };
+    }
+}
+// check: MoveSourceCompilerError resource type not copyable
```

Comment:
If we leave in this test, it needs to be an exp file. Or it needs to be moved out of functional tests

Reply:
Yes, I agree that this is not a functional test as this is testing a language property. Let me make an exp file first and then, we can decide where to put this test. (Same applies to the other two similar cases).

Reply:
Second thought, I think this test can be removed because it is essentially the same as `constraints_not_satisfied_struct_field.move` under `move-lang/tests/move-check/typing`, except that it is instantiated with libra-specific structs.

Any thoughts?

Reply:
I would just keep them as exp tests in the functional test suite. 

The move-check stuff should be stdlib agnostic

Reply:
Yes, I am aware that the tests in `move-check` are stdlib agonistic. I am suggesting to remove `event_no_resource.move` because it is essentially testing the same thing as `constraints_not_satisfied_struct_field.move`.

I also tried making it an exp file but the test infra is not very happy with the generated exp file because the file is compiled in a `.tmp****` directory and the `.tmp****` is contained in the error message too and changes on every run...

Reply:
I'd say kill these yeah. If you have any tests that are basically just type checker tests, they probably aren't that interesting. 

The tmp file thing might be an issue eventually... But for now since we are killing the test we can punt it

If they are, we can move them into move-check. No need to recheck 8 dozen times that constraints and function args work properly :)

Reply:
Got it! Thanks for the clarification!